### DISCUSSION TITLE
[Question] Hide mouse cursor on whole page

### DISCUSSION THREAD

**Discussion by azat385** [flet-dev/flet#2716] [Q&A]
How to set mouse_cursor.NONE to whole page?

**Comment by ndonkoHenri** [comment]
You can wrap all your page controls in a [`GestureDetector`](https://flet.dev/docs/controls/gesturedetector), then set the [`mouse_cursor`](https://flet.dev/docs/controls/gesturedetector#mouse_cursor) to `ft.MouseCursor.NONE`

**Comment by azat385** [comment]
> You can wrap all your page controls in a [`GestureDetector`](https://flet.dev/docs/controls/gesturedetector), then set the [`mouse_cursor`](https://flet.dev/docs/controls/gesturedetector#mouse_cursor) to `ft.MouseCursor.NONE`

GUI is made for touch screen of Raspberry pi, so i dont need the mouse cursor at all.

```python
import flet as ft

def main(page: ft.Page):
    page.add(ft.Text("Hello, world!"))

ft.app(main)
```
Can you provide an example for whole page please?
